client/src/main/java/io/pravega/client/stream/impl/StreamSegmentsWithPredecessors.java

```diff
@@ -26,8 +27,10 @@
 public class StreamSegmentsWithPredecessors {
     private final Map<Segment, List<Integer>> segmentWithPredecessors;
     private final Map<Integer, List<SegmentWithRange>> replacementRanges;
+    @Getter
+    private final String delegationToken;
```

Comment:
Again, I don't think this belongs here. This is a sub-stream data structure that is used in-memory on the client. I think this should be fixed in the caller.